### ISSUE TITLE
Ability to set file priority on folders and entire torrent

### ISSUE THREAD

**Issue by zp** [jfurrow/flood#91]
If you're downloading a torrent with a lot of files and/or directories, and you only want to download specific files, you currently have to set the priority for each individual file manually. (from what I can tell)

It would be nice to be able to mass set the file priorities for the entire torrent and/or the folders inside the torrent at once. 

That way you can easily set all files in the torrent/folder to "don't download", then manually tick the files you want to enable, for example.

**Comment by jfurrow** [comment]
I agree. I need to figure out a solid way of multi-selecting torrents. I'll work on this.

**Comment by jfurrow** [comment]
This is mostly done in https://github.com/jfurrow/flood/pull/125. I will likely merge it within the next 72 hours.

**Comment by jfurrow** [comment]
#125 finally merged :)

**Comment by KingDuckZ** [comment]
I'm a new user and I'm trying to do the same thing as @zp . I found the "select all" checkbox to be fairly intuitive and I thought by choosing "don't download" in the combo box that appears at the bottom I would disable all the selected files, however it's not working for me. I came here looking for a solution, and reading this thread it looks like I'm doing it right, but I'm unable to change anything on the selected items.

Unrelated, but I also noticed another strange behaviour: if you select some files (or all files even) and hit the 'download' button, the resulting tar contains all the files flattened (ie: no directories), and even some missing files.
